### DISCUSSION TITLE
Install asks me if i want to delete /usr/local

### DISCUSSION THREAD

**Discussion by VasilisThePikachu** [skycocker/chromebrew#10623] [Q&A]
I want to ask here first, why does the installer want to delete /usr/local?... like ALL of it? Having a quick look through it contains pretty important stuff.

Looking at the code I feel like something is going wrong
https://github.com/chromebrew/chromebrew/blob/9002f31d119f4d0dc5f312aee5421beed26ab4b9/install.sh#L53-L61

Reproducing on vt2 and crosh shell with version 126.0.6478.222
I did write the install command manually since I dont know if there is a way to "paste"

**Comment by VasilisThePikachu** [comment]
So I did check https://github.com/chromebrew/chromebrew/discussions/9757 right after this. However I feel like it should be mentioned that /usr/local is devinstall stuff expliticly when attempting to install or in the readme file.

**Comment by satmandu** [comment]
You are free to comment that line out and do an install!

This just avoids user support issues if there is a conflict with devinstall....

**Comment by adawdev** [comment]
There are several lines in the installer that do things like calling `mkdir` or `tar` using polite (non-overwrite) methods, this means that if a partial install left anything behind any follow up attempts will fail when these tools exit with error codes if the directory isn't removed first.

All of the real system binaries and libraries are installed in the immutable parts of the filesystem, such as `/bin` or `/usr/lib64` so that wiping `/usr/local` doesn't affect the main system or get blocked by write protection on the immutable partition.
